package.json

```diff
@@ -43,7 +43,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js"
   },
-  "version": "2.26.0",
+  "version": "2.26.1",
```

Comment:
This is gonna be confusing. We haven't released `2.26.0` yet, so no need to bump.

Reply:
Hmmm. I took from the meeting yesterday that every PR would update the version to some degree - but then, we're not automatically releasing yet, so fair point.